src/pages/settings/e-invoice/EInvoice.tsx

```diff
@@ -180,8 +245,9 @@ export function EInvoice() {
               handleChange('settings.e_invoice_type', value)
             }
             disabled={disableSettingsField('e_invoice_type')}
+            customSelector
           >
-            {isPeppolStandardEnabled && <option value="PEPPOL">PEPPOL</option>}
+            {shouldShowPEPPOLOption() && <option value="PEPPOL">PEPPOL</option>}
             <option value="FACT1">FACT1</option>
             <option value="EN16931">EN16931</option>
             <option value="XInvoice_3_0">XInvoice_3.0</option>
```

Comment:
showShow() breaks the app, removing customSelector fixes the issue.. need to look into this please

Reply:
@turbo124 The issue has been fixed in #2168 . The PR was created based on #2163 . Let me know your thoughts.

Reply:
@Civolilah cool, should we close this pr to prevent any other confusion?